### PR TITLE
Protobuf linter

### PR DESCRIPTION
**Pull request by dmathieu**
This sets up a linter for protobuf definitions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
     - run: make fmt
     - run: make gomodtidy
     - run: make fieldalignment
+    - run: make protolint
```
Comment (axw) [inline .github/workflows/ci.yml:21]: Would it make sense to replace all the above with `make all`?

**Review by axw — APPROVED** [review]
thank you :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dmathieu) [reply .github/workflows/ci.yml]: That makes sense (note: it also runs `generate-modelpb`, which we didn't run before).

Comment (axw) [reply .github/workflows/ci.yml]: :+1: that sounds desirable, it'll ensure that generated types are up to date.

**Pull request merged**